src/app/lib/utilities/escapeEntities/index.js

```diff
@@ -0,0 +1,26 @@
+// These are the commonly occuring characters, add more if need be.
```

Comment:
is it worth us using a pre-built utility for this? Something like `escape-html` is only 335B https://bundlephobia.com/result?p=escape-html@1.0.3

Keen to see what others might think, but using a pre-existing utility would remove the need for maintaining this ourselves as well as removing the tests.

Reply:
https://bundlephobia.com/result?p=escape-goat@3.0.0 is also great! 🐐

Reply:
Thanks @hindsc52 @simonsinclair I've changed to use `htmlUnescape` from `escape-goat`. Ran it against the previous test and it's working as expected.